app/controllers/volunteers_controller.rb

```diff
@@ -294,12 +293,7 @@ def home
   # and only in the region ids that are passed in
   # all super admins are filtered out for security purposes
   def unassigned_or_in_regions(admin_region_ids)
-    unassigned = Volunteer.not_super_admin.unassigned
-
-    volunteers_in_regions = Volunteer.not_super_admin
-                                     .assigned_to_regions(admin_region_ids)
-
-    volunteers_in_regions + unassigned
+    Volunteer.not_super_admin.assigned_to_regions(admin_region_ids) + Volunteer.not_super_admin.unassigned
```

Comment:
I think I may have squashed code from what we worked on before into scopes and this now needs to be merge conflict resolved.

Can you check that out or do you want me to?